src/esportsbot/cogs/MusicCog.py

```diff
@@ -427,31 +431,34 @@ async def reset_music_allowance(self):
         """
         self._time_allocation = defaultdict(lambda: self._allowed_time)
 
-    async def __setup_channel(self, ctx: Context, channel_id, arg):
+    async def __setup_channel(self, ctx: Context, channel_id: int, arg: str):
         """
         Sends the preview and queue messages to the music channel and adds the ids of the messages to the database.
         If the music channel is not empty and the correct arg is set, also clears the channel.
-        Args:
-            ctx: The context of the messages, used to send the messages to the channels.
-            channel_id: The id of the music channel
-            arg:
+        :param ctx: The context of the messages, used to send the messages to the channels.
+        :param channel_id: The id of the music channel.
+        :param arg: Optional arg to perform extra utilities while setting the channel up.
         """
+
+        # Get a discord object of the channel.
         channel_instance = self._bot.get_channel(channel_id)
         if channel_instance is None:
             channel_instance = await self._bot.fetch_channel(channel_id)
+
+        # Only need to get a few messages to check if the channel is non-empty.
         channel_messages = await channel_instance.history(limit=2).flatten()
         if len(channel_messages) > 1:
-            # If there are messages in the channel
+            # If there are messages in the channel.
             if arg is None:
-                await ctx.channel.send(
-                    "The channel is not empty, if you want to clear the channel for use, use !setmusicchannel -c <id>")
+                await ctx.channel.send("The channel is not empty, if you want to clear the channel for use, "
+                                       "use !setmusicchannel -c <channel>")
```

Comment:
Could do something like this in case the command prefix is changed:
```suggestion
                                       f"use {self._bot.command_prefix}setmusicchannel -c <channel>")
```